resources/main.js

```diff
@@ -21,9 +30,12 @@ $(function () {
     });
   });
 
-$(function () { document.getElementById("mw-scratch-confirmaccount-click-copy").onclick = function() {
```

Comment:
Is this just cleanup/robustness? It doesn't seem otherwise related to block expiration dates

Reply:
Fixes crash due to null access.